groovy/groovy.editor/src/org/netbeans/modules/groovy/editor/utils/GroovyUtils.java

```diff
@@ -236,4 +236,25 @@ private static String stripPackageFromTypeParams(String params) {
         }
         return sb.toString();
     }
+    
+    /**
+     * True, if the indexing is enabled. Depends system property or {@link #setIndexingEnabled(boolean)}.
+     * 
+     */
+    private static volatile boolean indexingEnabled = Boolean.valueOf(System.getProperty("org.netbeans.modules.groovy.editor.api.indexingEnabled", "true"));
```

Comment:
This property should be documented in _arch.xml_ file as `group="systemproperty"`.

Reply:
I considered the 'solution' a hack, so I didn't plan to document it  in arch document; otherwise I'd make a proper friend API call for it. After all, this hack should improve / change during 12.5 release - should we document it in the API/arch docs ?